.history/console/console-init/oauth-proxy/cfg/oauth-proxy-kubernetes_20200114170523.cfg

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2019, EnMasse authors.
+# License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+#
+
+## Oauth-Proxy Config File (Kubernetes)
+
+http_address = ""
+https_address = ":8443"
+provider = "oidc"
+oidc_issuer_url = "${ISSUER}"
+
+tls_cert_file = "/etc/tls/private/tls.crt"
+tls_key_file = "/etc/tls/private/tls.key"
+
+upstreams = [
+<<<<<<< HEAD
```

Comment:
seems you have unresolved merge conflicts, please fix that

Reply:
Okay, let me check.

Reply:
Updated the pull request.